### ISSUE TITLE
DnD mode from volume keys not working on LG X Venture?

### ISSUE THREAD

**Issue by randylewis7** [zacharee/SystemUITunerRedesign#19]
Hey, first off I want to say thanks so much for creating this app! I really hate the lack of ability of newer android phones to go into silent mode from pressing the volume down keys (among other things), and I can't fathom why the manufacturers tend to disable the built in SystemUI tuner. However, I am having some problems with getting the DnD mode feature to work on my phone, and was wondering if anyone had any suggestions.

I recently purchased a new LG X Venture, running android 7.0. I was very annoyed that it can't go into do not disturb mode from pressing the volume button down, so I looked into it, and discovered SystemUI Tuner. The built in version is apparently disabled on my phone (thanks LG), so I downloaded the version being discussed here  (https://play.google.com/store/apps/d...tuner&hl=en_US). My phone is not rooted, so I had to enable the permissions the app needs with adb. I followed the instructions on the app and also here (https://www.howtogeek.com/318539/how...stock-devices/) to get adb working with my phone and windows 7 computer. Eventually, everything seemed to work, and I typed in the three adb permission lines while connected over adb. I didn't get any output from any of the lines, but no errors either, so I would guess they worked. I then proceeded in the app, and started playing around with the settings. Among other things, I disabled a lot of icons in the phone status bar (as well as enabling auto detect), enabled extreme night mode, and enabled "DND Switch in Vol Slider".

The night mode definitely changed the tint of my screen, as soon as I clicked it. But the other two settings don't seem to have changed anything, even after a restart. My taskbar is still the same, with icons that I tried to hide still there. And most importantly, the volume button still doesn't put the phone into DnD mode. I hold it down until the phone is on vibrate, then I click it down again, and nothing happens. I can enable DnD mode from the taskbar, but then I can't even exit it from the volume keys.

Any idea what might be wrong? Is this just an inherent problem with my phone/the app, or did something go wrong enabling permissions? I would greatly prefer not to root my phone, but will consider it if nothing else works.

**Comment by zacharee** [comment]
That switch doesn't allow you to enable DND with the volume keys. It adds
a DND toggle to the volume dialog. As for hiding icons, if they don't hide,
they don't hide.

Sent from Astro <https://www.helloastro.com> for Android

On 5/12/18 at 2:33 PM, randylewis7 wrote:

Hey, first off I want to say thanks so much for creating this app! I really
hate the lack of ability of newer android phones to go into silent mode
from pressing the volume down keys (among other things), and I can't fathom
why the manufacturers tend to disable the built in SystemUI tuner. However,
I am having some problems with getting the DnD mode feature to work on my
phone, and was wondering if anyone had any suggestions.

I recently purchased a new LG X Venture, running android 7.0. I was very
annoyed that it can't go into do not disturb mode from pressing the volume
button down, so I looked into it, and discovered SystemUI Tuner. The built
in version is apparently disabled on my phone (thanks LG), so I downloaded
the version being discussed here (
https://play.google.com/store/apps/d...tuner&hl=en_US). My phone is not
rooted, so I had to enable the permissions the app needs with adb. I
followed the instructions on the app and also here (
https://www.howtogeek.com/318539/how...stock-devices/) to get adb working
with my phone and windows 7 computer. Eventually, everything seemed to
work, and I typed in the three adb permission lines while connected over
adb. I didn't get any output from any of the lines, but no errors either,
so I would guess they worked. I then proceeded in the app, and started
playing around with the settings. Among other things, I disabled a lot of
icons in the phone status bar (as well as enabling auto detect), enabled
extreme night mode, and enabled "DND Switch in Vol Slider".

The night mode definitely changed the tint of my screen, as soon as I
clicked it. But the other two settings don't seem to have changed anything,
even after a restart. My taskbar is still the same, with icons that I tried
to hide still there. And most importantly, the volume button still doesn't
put the phone into DnD mode. I hold it down until the phone is on vibrate,
then I click it down again, and nothing happens. I can enable DnD mode from
the taskbar, but then I can't even exit it from the volume keys.

Any idea what might be wrong? Is this just an inherent problem with my
phone/the app, or did something go wrong enabling permissions? I would
greatly prefer not to root my phone, but will consider it if nothing else
works.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/zacharee/SystemUITunerRedesign/issues/19>, or mute the
thread
